### ISSUE TITLE
Reflow tests to be written

### ISSUE THREAD

**Issue by HelenBurge** [auto-wcag/auto-wcag#2424]
I have been working on the WCAG 3 Text Reflow ACT Rules, and created 3 tests for 2.1.

I propose adding them to the ACT Rules.

https://docs.google.com/document/d/1ucdwQdwGCuTPzKXAfF8tTe5RRXsWVFE2bB--6zFRjiQ/edit?tab=t.nvxjvvpod5a5

**Comment by G0dwin** [comment]
I would find it easier to read through the Netify preview but it's failing. I'm not sure what is going on exactly, it looks like a problem with the script. @WilcoFiers is this something that you have seen?

In the meantime I'll look through the markown source.
